core/tests/puppeteer-acceptance-tests/blogpostAdmin-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,20 @@
+const e2eBlogPostAdmin = require("../utility-functions/blogPostAdminUtils.js");
+const testConstants = require("../utility-functions/testConstants.js");
+
+const blogDashboardUrl = testConstants.URLs.BlogDashboard;
+const homePage = testConstants.Dashboard.MainDashboard;
+
+async function deleteDraftAsBlogPostAdmin() {
+  const blogPostAdmin = await new e2eBlogPostAdmin();
+  await blogPostAdmin.openBrowser();
+
+  await blogPostAdmin.signInWithEmail("testadmin@example.com");
+  await blogPostAdmin.waitForPageToLoad(homePage);
+  await blogPostAdmin.goto(blogDashboardUrl);
+  await blogPostAdmin.createDraftBlogPostByTitle("Test Blog Post");
+  await blogPostAdmin.deleteDraftBlogPostByTitle("Test Blog Post");
```

Comment:
I think you need to have some predicates for expectations. E.g. blogPostAdmin.expectNumberOfBlogPostsToEqual(0) or blogPostAdmin.expectNumberOfDraftBlogPostsToEqual(0), etc.

These encode what the person "sees". 

Also, for the purposes of this test -- let's actually add a couple more lines where the blog post admin goes to the "blog viewer page" and verifies that there is 0 posts or 1 post. I want to see how you handle the case whre there are multiple different pages in play -- it should not be possible to call createDraftBlogPostByTitle on the blog viewer page, say. See if you can figure out a pattern for that which makes things clear (and feel free to ask if you need help)!